pkg/apis/core/v1alpha1/deployer_lifecycle.go

```diff
@@ -89,6 +92,10 @@ func (ds *DeployerStatus) PropagateServiceStatus(ss *corev1.ServiceStatus) {
 	deployerCondSet.Manage(ds).MarkTrue(DeployerConditionServiceReady)
 }
 
+func (ds *DeployerStatus) ServiceNameCollision(collisionKind string, collisionName string) {
```

Comment:
methods that manipulate a condition typically take the form `Mark{kind}NotOwned`

See https://github.com/projectriff/system/blob/v0.4.x/pkg/apis/core/v1alpha1/deployer_lifecycle.go#L56-L59

Reply:
So would you prefer something like `MarkServiceNameInUse`?